src/config.js

```diff
@@ -253,3 +247,18 @@ export const DRAWING_HIT_TOLERANCE = 6
 export const VECTOR_TILES_STYLE_ID = 'ch.swisstopo.leichte-basiskarte_world.vt'
 
 export const VECTOR_TILES_STYLE_URL = `https://vectortiles.geo.admin.ch/styles/${VECTOR_TILES_STYLE_ID}/style.json`
+
+/**
+ * Display a big developpment banner on all but these hosts.
+ *
+ * @type {String[]}
+ */
+ export const NO_WARNING_BANNER_HOSTNAMES = ['map.geo.admin.ch']
+
+ /**
+ * Display a warning ribbon ('TEST') on all but these hosts.
```

Comment:
Comment should be updated: `Display a warning ribbon ('TEST') on all these hosts.`

Reply:
OK. Nice catch.